### ISSUE TITLE
BUG: np.reciprocal on integers uses C division, not true division

### ISSUE THREAD

**Issue by eric-wieser** [numpy/numpy#10374]
This is documented correctly

> For integer arguments with absolute value larger than 1 the result is always zero because of the way Python handles integer division. For integer zero the result is an overflow.

But seems useless and undesirable.

**Comment by mhvk** [comment]
The docstring is arguably incorrect for python3. Type promotion as for `true_divide` seems reasonable (and here I would expect less resistance than for `power`!).

**Comment by rgommers** [comment]
Agreed, we should change this. I think the "Bug" qualification is fair for Python 3.x.

**Comment by ilemhadri** [comment]
@rgommers this bug is still active, and I may be able to contribute. can I propose a PR?

**Comment by rgommers** [comment]
@ilemhadri no need to ask, you can work on any open issue and submit a PR for it.
